week5.ipynb

```diff
@@ -0,0 +1,366 @@
+{
```

Comment:
<p>same comment as above for question 2 - what they create is a dictionary, not a list</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-jolsenvaldez/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>